Chunk 1: Correct wrong cross reference (also submitted upstream).
Chunks 2 and 3(3): Do not report contractions by default (local change).
Chunks 3(1) and 4: Do not use /usr/local/etc/ (local change).
Chunk 3(2): Correct link to Debian wordlist (agreed with upstream).

Index: igor.1
--- igor.1.orig
+++ igor.1
@@ -30,7 +30,7 @@ to detect a file's type and only runs the applicable t
 Files compressed with
 .Xr gzip 1
 or
-.Xr bzip 2
+.Xr bzip2 1
 are automatically decompressed.
 If multiple files are specified, filenames are displayed before each
 is analyzed.
@@ -104,7 +104,7 @@ When ready for the actual commit, run
 .Nm
 without this option to check the date.
 Equivalent to
-.Fl abefmrsuw .
+.Fl abefmrsw .
 .It Fl z
 Run all applicable non-whitespace
 .Pq content
@@ -154,15 +154,21 @@ documents.
 Additional files of misspelled words may be added with the
 .Ev IGORSPELLFILES
 environment variable, or placed in the
-.Pa /usr/local/etc/igor/spelling
+.Pa /etc/igor/spelling
 directory.
 The first word on each line, including apostrophes, will be included in the
 list of known misspellings to be checked.
 Blank lines and lines starting with a # character are ignored.
 The Debian project has a list of words at
-.Li http://anonscm.debian.org/gitweb/?p=lintian/lintian.git;a=blob_plain;f=vendors/debian/ftp-master-auto-reject/data/spelling/corrections;hb=HEAD
+.Lk https://salsa.debian.org/lintian/lintian/-/raw/master/data/spelling/corrections
 .It Fl u
 Identify contractions, which make translation more difficult.
+This test is off by default on
+.Ox
+because
+.Ox
+documentation maintainers regard contractions as a normal feature
+of the English language and see no need to avoid them.
 .It Fl w
 Find general whitespace errors, including trailing whitespace at
 the end of lines, blank lines containing whitespace, and tabs
@@ -271,8 +277,8 @@ A list of whitespace-delimited filenames containing ad
 words.
 .El
 .Sh FILES
-.Bl -tag -width /usr/local/etc/igor/spelling/
-.It Pa /usr/local/etc/igor/spelling/
+.Bl -tag -width /etc/igor/spelling/
+.It Pa /etc/igor/spelling/
 Directory for additional misspelled word files.
 .El
 .Sh EXAMPLES
